node_modules/oae-core/about/css/about.css

```diff
@@ -0,0 +1,62 @@
+
+#about-description .about-description-empty,
+#about-description .about-description-empty a {
+    color: #888;
+}
+
+#about-description .about-description-empty a {
+    font-style: italic;
+    text-decoration: underline;
+}
+
+#about-members {
+    margin-top: 35px;
+}
+
+#about-members > div.about-show-all {
+    margin-top: 15px;
+}
+
+#about-members .about-members {
+    display: -webkit-box;
+    display: -moz-box;
+    display: -ms-flexbox;
+    display: -webkit-flex;
+    display: flex;
+    width: 180px;
+    -webkit-flex-wrap: wrap;
+        -ms-flex-wrap: wrap;
+            flex-wrap: wrap;
+}
+
+#about-members .about-members div.oae-thumbnail {
+    border: none;
+    height: 60px;
+    width: 60px;
+    -webkit-box-flex: none;
+       -moz-box-flex: none;
+        -webkit-flex: none;
+            -ms-flex: none;
+                flex: none;
+}
+
+/* Clear the border radius on base oae-thumbnail style */
+#about-members .about-members div.oae-thumbnail,
+#about-members .about-members div.oae-thumbnail a,
+#about-members .about-members div.oae-thumbnail div {
+    -webkit-border-radius: 0;
+       -moz-border-radius: 0;
+            border-radius: 0;
+}
+
+/* IE9 adds 5px of space after the images for no reason. Give the individual thumbnails a negative
```

Comment:
I'm pretty sure this behavior will go away once we remove all of the `box-flex` and other `flex` things in favor of something like `inline-block`